css/style.css

```diff
@@ -0,0 +1,10 @@
+#nav-bar{
+    position: fixed;
+    text-align: left;
+    top: 0;
+    width:100;
+    display:flex;
+}
+.nav-link{
+  display:flex;
+}
```

Comment:
A new line at the end of the CSS file must exist. Otherwise, a linter error will occur.